src/schema_compile.c

```diff
@@ -1547,9 +1547,6 @@ lys_recompile(struct ly_ctx *ctx, ly_bool log)
             lysc_module_free(mod->compiled);
             mod->compiled = NULL;
         }
-
-        /* free precompiled iffeatures */
-        lys_free_feature_iffeatures(mod->parsed);
```

Comment:
commit message says "unused functions", but this function is used, why it can be removed now?

Reply:
Yes, the message should be improved now.